packages/react/src/Card/Card.test.tsx

```diff
@@ -140,52 +141,50 @@ describe('Card', () => {
   })
 
   it('renders the label correctly into the document', () => {
-    const mockTestId = 'test'
+    const mockTestId = 'label'
     const classToCheck = 'Card__label'
 
     const {getByTestId} = render(
-      <Card href={mockHref} data-testid={mockTestId}>
-        <Card.Label>{mockLabel}</Card.Label>
+      <Card href={mockHref}>
+        <Card.Label data-testid={mockTestId}>{mockLabel}</Card.Label>
         <Card.Heading>{mockHeading}</Card.Heading>
         <Card.Description>{mockDescription}</Card.Description>
       </Card>,
     )
 
-    const cardEl = getByTestId(mockTestId).firstChild
```

Comment:
These tests have been updated as `firstChild` now always returns the heading, which isn't what these tests expect.